crates/biome_cli/tests/cases/assists.rs

```diff
@@ -77,10 +76,10 @@ fn assist_writes() {
         r#"{ "zod": true, "lorem": "ipsum", "foo": "bar" }"#.as_bytes(),
     );
 
-    let result = run_cli(
-        DynRef::Borrowed(&mut fs),
+    let (fs, result) = run_cli(
+        fs,
         &mut console,
-        Args::from([("check"), "--write", file.as_os_str().to_str().unwrap()].as_slice()),
+        Args::from(["check", "--write", file.as_os_str().to_str().unwrap()].as_slice()),
```

Comment:
Note: I ran a search-and-replace on the args to remove these odd parentheses. My refactor had to touch all these call sites anyway, so I figured it was a good time to do this. This probably resulted in most of the reduction in line count in this PR.